docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -39,6 +39,12 @@ of the project and those will go untested.
 To gain the ability to execute all aspects of the TinkerPop build system, other environmental configurations must be
 established. Those prerequisites are defined in the following subsections.
 
+As of TinkerPop 3.5.5, environment configuration for Gremlin Language Variants can be optional, as Docker becomes a system
+requirement to build and test GLVs inside of Maven. Docker enables the building and testing of the GLVs without the need to install GLV-specific environments.
```

Comment:
formatting is a bit inconsistent

Reply:
Will keep consistency with current document